### PR TITLE
fix initial color of user bounding boxes

### LINKED ISSUE #4699: initial color in viewport when adding a user bbox does not match color in menu
…becomes correct when you change the color manually.

### PR DESCRIPTION
**Pull request by MichaelBuessemeyer**
This PR fixes that the initial displayed color of a user bounding box in the settings is not correct.

### URL of deployed dev instance (used for testing):
- https://fixinitialuserbbcolor.webknossos.xyz

### Steps to test:
- Open a dataset in view mode and add a user bounding box. The color of the box in the settings should match the color displayed in the viewports.

### Issues:
- fixes #4699

------
- ~~[ ] Updated [(unreleased) changelog](../blob/master/CHANGELOG.unreleased.md#unreleased)~~
- ~~[ ] Updated [(unreleased) migration guide](../blob/master/MIGRATIONS.unreleased.md#unreleased) if applicable~~
- ~~[ ] Updated [documentation](../blob/master/docs) if applicable~~
- ~~[ ] Adapted [wk-connect](https://github.com/scalableminds/webknossos-connect) if datastore API changes~~
- ~~[ ] Needs datastore update after deployment~~
- [x] Ready for review

### CODE REVIEW

File: frontend/javascripts/libs/utils.js [hunk 1]
```diff
@@ -151,7 +151,8 @@ export function colorObjectToRGBArray({ r, g, b }: ColorObject): Vector3 {
 }
 
 export function getRandomColor(): Vector3 {
-  const randomColor = [0, 1, 2].map(() => Math.random());
+  // Here we generate three values between 0 and 1 that multipled with 255 will be integers.
```
Comment (daniel-wer) [inline frontend/javascripts/libs/utils.js:154]:
```suggestion
  // Generate three values between 0 and 1 that multiplied with 255 will be integers.
```

**Comment by MichaelBuessemeyer** [comment]
> Do you know why the color difference was so visible?

The `UserBoundingBoxInput` uses the `Utils.rgbToHex` function. I guess this causes the color to be completely wrong. I'll take a quick look and look whether I can also fix the bug in this method.

**Comment by MichaelBuessemeyer** [comment]
@daniel-wer Could you please give the newest changes a quick check again? :smile:

**Comment by daniel-wer** [comment]
Works very well :rocket:

**Pull request merged**